### ISSUE TITLE
Can't get iCloud sync to work

### ISSUE THREAD

**Issue by eyelessjerry** [glushchenko/fsnotes#1592]
### Description

Can't get iCloud sync to work – no matter how I try to deactivate/reactivate etc. Sometimes I can manually upload/download a file. I think iCloud is not much of a serious solution for syncing between iOS and macOS – not aware it has ever worked well for anyone with anything ... Dropbox usually works though.

### To Reproduce

1. Do whatever ... nothing helps.

### Expected behavior

that it might work

### FSNotes version

latest

### macOS/iOS version

latest stable build of Ventura for Apple Silicon

**Comment by gingerbeardman** [comment]
It has worked for us for 6 versions of FSNotes, and FSNotes team use it every day.

You don't have to do anything, there is no activation. FSNotes default folder is in iCloud so it should simply work out of the box.

Have you made some changes.

**Comment by eyelessjerry** [comment]
Yes, it seems to synchronize now after restart of the Mac … I did install FSNotes first on macOS to test it, then changed location to use iCloud (where a folder was not created automatically so I had to move files into an FS-folder later. Then today bought the FSNotes iOS app and it read the two notes I had created from the Mac but a new note I created to test the non-working (seemingly) Soulver integration did not sync over, but upon restart of the computer it did! (still confused about the formats of the files – never really uses md or markdown, but understand it as I use html, but basically needed the program to sync plain text notes between the units as it is not good enough to only use formatted notes like in Apple Notes – was aiming at handling bookmarks in a browser/plugin independent way and then you do not (normally) want clickable links etc.
